src/main/java/frc/robot/subsystems/endgame/EndgameSparkMaxIO.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.subsystems.endgame;
+
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkBase.IdleMode;
+import com.revrobotics.CANSparkLowLevel.MotorType;
+
+import frc.robot.Constants.EndgameConstants;
+import frc.robot.subsystems.endgame.EndgameSubsystem;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+
+public class EndgameSparkMaxIO implements EndgameIO{
```

Comment:
The interface functions will need to be added to this class (exactly like you did on EndgameSimIO already)